package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "integreat-webapp",
-  "version": "2019.12.0",
+  "version": "0",
```

Comment:
Is this a valid version? I think npm is not able to parse this as this is not in the spec. Should be 0.1